pkg/commands/service.go

```diff
@@ -15,24 +15,24 @@ type Service struct {
 	ID            string
 	OSCommand     *OSCommand
 	Log           *logrus.Entry
-	Container     *Container
+	Container     []*Container
```

Comment:
if we are making this an array (which I think is a good idea) we should call it the plural `Containers`

Reply:
yeah - the pain horror, is that i'm using them as though there is only one so we need to figure out some UX

Reply:
One solution could be to allow hitting a key in the services panel (like space) that would swap out the contents of the services panel with the list of containers for the selected service. Then flicking through those would display the logs for each one etc, and maybe we could make it that if you hit space on one of those containers, it becomes the selected container for that service, meaning any commands run on the service that involve a single container will affect that container (e.g. attach). 

I think however that commands like stop/restart should affect all containers of a service, and if a user wants to be more specific they can hit space on a service and then run the specific command on the container.

Obviously the default selected container for a service would be the first in the list, meaning most users won't even notice the extra complexity which I think is good.

As for how to implement this, I do know that in lazygit (a similar application) I made it that you can drill down into the files of a commit by pressing 'enter' on the commit (though in lazydocker 'enter' is more for focusing the main panel). I have a bad feeling that I wrote that code in a fairly ad-hoc way, but I might be wrong!

Ideally we'd want some way to easily hook into the existing functionality of the current container's panel when you're viewing the containers of a service. I'm not sure how best to go about that though.

What are your thoughts?

Reply:
(by the way, we don't need to support the whole multiple-containers-per-service thing in this PR if we don't need to, we can make a separate issue for it)

Reply:
yeah - that might be the right thing - I need to clean up some things, i'll ping when i think its in a not-so breaking state (I don't use compose so I havn't a good test set for that.)